src/vs/editor/contrib/markdownRenderer/browser/markdownRenderer.ts

```diff
@@ -23,6 +24,7 @@ export interface IMarkdownRenderResult extends IDisposable {
 export interface IMarkdownRendererOptions {
 	editor?: ICodeEditor;
 	codeBlockFontFamily?: string;
+	additionalClasses?: string[];
```

Comment:
Is `additionalClasses` needed or is this something that callers could add if they need it?